app/views/objects/show/_methods.html.slim

```diff
@@ -0,0 +1,29 @@
+div class="md:px-3 my-3"
+  h3 class="lg:text-2xl text-lg text-gray-900 dark:text-gray-200 font-semibold"
+    = title
+  hr
+- methods.sort_by(&:name).each do |m|
+  div class="md:p-3 py-3 my-3"
+    a style="display: block; position: relative; top: -80px; visibility: hidden;" id="#{method_anchor m}"
+    div class="flex flex-wrap"
+      div class="w-full md:w-10/12"
+        - if m.call_sequence.empty?
+          h4 class="lg:text-xl text-md text-gray-900 dark:text-gray-200 font-semibold"
+            a href="##{method_anchor(m)}"
+              = m.name
+        - else
+          a href="##{method_anchor(m)}"
+            - m.call_sequence.each do |seq|
+              h4 class="lg:text-xl text-md text-gray-900 dark:text-gray-200 font-semibold"
+                = seq
```

Comment:
Take a note: method names become smaller (constant name has `3xl` size, methods had `2xl` size, I don't know a size between them, so I've assigned `2xl` size to headers like "Class Methods" and change methods headers to `xl`).